app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -35,10 +44,24 @@
     ProgressBar progressBar;
     @BindView(R.id.toolbar)
     Toolbar toolbar;
+    @BindView(R.id.checkboxEnableUrl)
+    CheckBox checkBoxEnableUrl;
+    @BindView(R.id.addUrlContainer)
+    View addUrlContainer;
+    @BindView(R.id.etBaseUrl)
+    EditText etBaseUrl;
 
     @Inject
     ILoginPresenter presenter;
 
+    @Inject
+    HostSelectionInterceptor interceptor;
+
+    private String DEFAULT_BASE_URL;
+
+    public static final String BASE_URL_DEBUG = "https://open-event-dev.herokuapp.com/api/v1/";
+    public static final String BASE_URL_RELEASE = "https://www.eventyay.com/api/v1/";
```

Comment:
No need to add logic and change it this way. You can add debug and release version strings in build.gradle and just use `BuildConfig.BASE_URL`